source/manuals/programming-languages/js.html.md.erb

```diff
@@ -126,57 +126,63 @@ If you're using [Prettier](#prettier), this will be set up for you. Otherwise, y
 
 ## Naming conventions
 
-* Avoid single letter names. Be descriptive with your naming.
+Follow the following conventions when naming symbols in your JavaScript code.
 
-  ```javascript
-  // Bad
-  var n = 'thing'
-  function q () { ... }
-
-  // Good
-  var name = 'thing'
-  function query () { ... }
-  ```
+**Why:** This lets future developers know how to interact with objects and sets the appropriate affordances. It also follows the conventions of the standard library.
```

Comment:
Nitpick: is there an alternative word we could use to `affordances`? I had to look up what it means, maybe we could use something more commonly used?

Reply:
Good shout, I copied that from the previous content without thinking further. I'll rephrase.